app/views/static_pages/sponsors.html.erb

```diff
@@ -6,18 +6,22 @@
 		<p><b><span class="small_title">Verizon's International Student Program</span></b><br />
 			iPhone 6S or 6S Plus available for pre-order now! Trade in your old smartphone for a new iPhone 6S or 6S Plus and get up to $400, when you trade in your old iPhone or Android, and another $100 when you switch to Verizon. Visit the Verizon store at 10250 Baltimore Ave, College Park, MD 20740 to learn more about this offer and <a href="http://vzw.com/internationalstudent">International Student Program</a>. No deposit. Simple signup. Just bring your international passport and student visa.<br /> <br />
 			<b><span class="small_title">关于Verizon International Student Program</span></b><br />
-				即日起，Verizon接受优先预订 iPhone 6S 及 iPhone 6S Plus。折旧你的旧iPhone或其他手机换新iPhone 6S 或 6S Plus可获享高达$400的折旧换新退款。更多详情，请前往Verizon 10250 Baltimore Ave, College 了解优惠或留学生计划 -- 开新号无需押金，简单申请，请带上护照和留学签证。若已有美国电话号码，转号到Verizon可获得$100账单减免</p>
+				即日起，Verizon接受优先预订 iPhone 6S 及 iPhone 6S Plus。折旧你的旧iPhone或其他手机换新iPhone 6S 或 6S Plus可获享高达$400的折旧换新退款。更多详情，请前往Verizon 10250 Baltimore Ave, College 了解优惠或留学生计划 -- 开新号无需押金，简单申请，请带上护照和留学签证。若已有美国电话号码，转号到Verizon可获得$100账单减免</p><br /> <br />
 	</div>
 	<div id="verizon_logo" class="col-md-4">
 		<%= link_to image_tag('verizon_logos/verizon_logo_for_web.png', :class=>"img-responsive"), "http://www.verizon.com" %>
 	</div>
 </div>
-<div id="smallSponsorRow" class="row sponsorsContent">
-	<p><b><span class="small_title">We're also sponsored by: </span></b></p>
+<div id="largeSponsorRow" class="row sponsorsContent">
```

Comment:
which id do you mean? and what problem the css styles cause?

On Fri, Aug 26, 2016 at 11:41 PM, Andrew Gu <notifications@github.com>
wrote:

> In app/views/static_pages/sponsors.html.erb
> <https://github.com/andrewgu12/CSA/pull/7#discussion_r76509464>:
>
> >  	</div>
> >  	<div id="verizon_logo" class="col-md-4">
> >  		<%= link_to image_tag('verizon_logos/verizon_logo_for_web.png', :class=>"img-responsive"), "http://www.verizon.com" %>
> >  	</div>
> >  </div>
> > -<div id="smallSponsorRow" class="row sponsorsContent">
> > -	<p><b><span class="small_title">We're also sponsored by: </span></b></p>
> > +<div id="largeSponsorRow" class="row sponsorsContent">
>
> @lchen110 <https://github.com/lchen110> if you change the id, make sure
> also to update the css styles!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/andrewgu12/CSA/pull/7/files/f663e6fe0081cba92cc8105aea64c9cac172f95a#r76509464>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ARPgyCi7eneny665oscxCk6O_yMUbdqSks5qj7HqgaJpZM4Jumlv>
> .
>